src/html/get-metadata.js

```diff
@@ -19,7 +19,7 @@ const {
 
 function yaml(section) {
   const yamls = selectAll('yaml', section);
-  section.meta = obj(flat(map(yamls, ({ payload }) => payload)));
+  section.meta = Object.assign({}, obj(flat(map(yamls, ({ payload }) => payload))));
```

Comment:
Why the additional `Object.assign`?

Reply:
👍 Some left-overs from testing various approaches. I'll definitely remove in the final PR.